homeassistant/components/shelly/sensor.py

```diff
@@ -464,6 +464,78 @@ class RestSensorDescription(RestEntityDescription, SensorEntityDescription):
         device_class=SensorDeviceClass.ENERGY,
         state_class=SensorStateClass.TOTAL_INCREASING,
     ),
+    "total_act": RpcSensorDescription(
+        key="emdata",
+        sub_key="total_act",
+        name="Total active energy",
+        native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
+        value=lambda status, _: round(float(status) / 1000, 2),
+        device_class=SensorDeviceClass.ENERGY,
+        state_class=SensorStateClass.TOTAL_INCREASING,
+    ),
+    "a_total_act_energy": RpcSensorDescription(
+        key="emdata",
+        sub_key="a_total_act_energy",
+        name="Phase A total active energy",
```

Comment:
I would make this diagnostic entity category. It's to see if Total Active Energy is correct.

Reply:
Changed to disabled by default